pkg/operator/staticpod/installerpod/cmd.go

```diff
@@ -348,6 +351,10 @@ func (o *InstallOptions) copyContent(ctx context.Context) error {
 		return err
 	}
 
+	// remove the existing file to ensure kubelet gets "create" event from inotify watchers
+	if err := os.Remove(path.Join(o.PodManifestDir, podFileName)); err == nil {
```

Comment:
we could log a potential error by checking `err != nil && !os.IsNotExist(err)`

Reply:
since we know there's a potential err we could simply return the error and retry at a later time, wdyt?

Reply:
@p0lyn0mial right, the pod will retry... and if we can't remove the existing file it is not likely we would be able to write new content :)